vizro-core/src/vizro/_vizro.py

```diff
@@ -32,7 +32,13 @@ def __init__(self, **kwargs):
                 [Dash documentation](https://dash.plotly.com/reference#dash.dash) for possible arguments.
 
         """
-        self.dash = dash.Dash(**kwargs, use_pages=True, pages_folder="", title="Vizro")
+        self.dash = dash.Dash(
+            **kwargs,
+            pages_folder="",
+            suppress_callback_exceptions=True,
+            title="Vizro",
+            use_pages=True,
+        )
```

Comment:
> Should we allow users to overwrite these arguments?

No, because they are all (apart from `title`) essential to the functioning of Vizro. And the `title="Vizro"` here gets overwritten in the `build` method so its effect is very minor.

> If not, should we at least explain them somewhere in the docs?

Also no, because no user will care 😅

The only possible exception to this is the `suppress_callback_exceptions` where we're possibly suppressing too much, but let's just note that in a comment (see suggestion).